pom.xml

```diff
@@ -16,16 +16,11 @@
   <name>guacamole: variant caller</name>
 
   <properties>
-    <adam.version>0.19.0</adam.version>
```

Comment:
I'm going with the heuristic that if it's used twice, it should be factored out, otherwise it should just be on the dependency-declaration.

Factoring out versions that only appear once seems more verbose and less clear, to me.

lmk if you disagree.